<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewGraphDialog</class>
 <widget class="QWidget" name="CreateNewGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utworzenie nowego grafu</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>13</y>
     <width>202</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="graphNameLabel">
      <property name="text">
       <string>Nazwa grafu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="graphNameText"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="OrderLabel">
      <property name="text">
       <string>Rodzaj</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="orderComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="weightLabel">
      <property name="text">
       <string>Wagi</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="weightComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
